.github/actions/integration-test/action.yml

```diff
@@ -0,0 +1,48 @@
+name: 'integration-test'
+description: 'Runs an integration test and uploads the results'
+author: 'dd'
+inputs:
+  test-server:
+    description: 'Test runner server specification file'
+    required: true
+    default: 'workspace-wsmtest.json'
+  test:
+    description: 'Test suite or config to run'
+    required: true
+    default: 'suites/FullIntegration.json'
+  vault-token:
+    description: 'Vault token for rendering'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Set permissions and render test configurations
```

Comment:
Do we need to render configs for the service in order to run integration tests? I thought the rendering was only required for running the service, and integration tests only use the client.

Reply:
I believe it is required, but I am not sure. It was in the existing actions so I assumed it was needed.
I made PF-744 suggesting that we combine the render in `integration` and the render in `service` into a single tool that both use. When we do that, we won't care if this was needed or not.

Reply:
My understanding is that it's required if we're running a local server (which we do for PR tests) but shouldn't be otherwise. sgtm doing this as part of a larger config refactor

Reply:
I think we should assign the work to @jaycarlton to give him an excuse to play with the slack notification! :)